airflow/utils/db.py

```diff
@@ -644,7 +644,7 @@ def create_default_connections(session: Session = NEW_SESSION):
         Connection(
             conn_id="tabular_default",
             conn_type="tabular",
-            host="https://api.tabulardata.io/ws/v1",
+            host="https://api.tabular.io/ws",
```

Comment:
I assume this change cause issues with
https://github.com/apache/airflow/blob/bfcae349b88fd959e32bfacd027a5be976fe2132/tests/providers/tabular/hooks/test_tabular.py#L29

Reply:
Thanks, you are too kind 👍